docs/cardano_lock.md

```diff
@@ -0,0 +1,57 @@
+# Cardano Lock
```

Comment:
Rewrite this document according to the proposal: https://www.notion.so/cryptape/CKB-AUTH-f70ec65bbbaf45acaf13e67d5f4799a2